Code/Tools/AssetProcessor/native/utilities/ApplicationManagerBase.cpp

```diff
@@ -894,6 +894,10 @@ void ApplicationManagerBase::Destroy()
     delete m_assetRequestHandler;
     m_assetRequestHandler = nullptr;
 
+    // Destroy file monitor early so now new events get generated to propogate through
```

Comment:
nitpick: typo: now -> no, propogate -> propagate
```suggestion
    // Destroy file monitor early so no new events get generated to propagate through
```

Reply:
whoops!  I'll fix this typo